tests/testthat/test-renderer1-facet-lines.R

```diff
@@ -47,7 +47,7 @@ wb.facets <-
        selector.types=list(country="multiple"),
        title="World Bank data (multiple selection, facets)")
 
-remDr$setTimeout(milliseconds=60 * 1000 * 2)
+Sys.sleep(60 / 1000)
```

Comment:
I think the old code `remDr$setTimeout` is trying to set the timeout duration, it's the time the remote driver will wait for an operation to be completed. On the other hand, `Sys.sleep` is just simply pause. Any specific reason to update it this way?

Reply:
I think the default_timeout is the equivalent in chromote https://rstudio.github.io/chromote/reference/Chromote.html#public-fields

Reply:
> https://rstudio.github.io/chromote/reference/Chromote.html#public-fields

@tdhock thank you for pointing that out. I changed the system sleep command to default_timeout.